.vuepress/public/_redirects

```diff
@@ -155,3 +155,6 @@
 
 # Remove Feature reference
 /reference/features/                                /reference/
+
+# Remove Installation page
+/learn/getting_started/installation.html        /learn/getting_started/quick_start.html
```

Comment:
Could you add a redirect for the removed "What's next" page, please? It can just redirect to the quick start